src/main/resources/public/raml/resourceTypes/failed-notifications.raml

```diff
@@ -1,11 +1,11 @@
-description: This class searches for missed notifications in the database.
+description: This class searches for failed notifications in the database.
 get:
-  description: This method returns missed notification(s) given a subscription name.
+  description: This method returns failed notification(s) given a subscription name.
```

Comment:
It should be "given one or several subscription names" right?

Reply:
In story the old and new end points differ in name only...
[old-----> /queryMissedNotifications?subscriptionName=Sub1,Sub2,Sub3 AND new---->/failed-notifications?subscriptionNames=Sub1,Sub2,Sub3]

 so I changed the name only. However, I think it can take only one subscription at a time.

Reply:
You're right, I was a bit confused since we've just assumed this behavior in the issue. I don't know where we got confused but the issue #346 and #291 describes the option of retrieving all failed notifications for several subscriptions. Ignore my comment on the logging then and keep it as is for now - but I''ll update the original issue so we can discuss the wanted behavior for this endpoint.